src/ape/api/accounts.py

```diff
@@ -98,6 +99,8 @@ def call(
             txn (:class:`~ape.api.transactions.TransactionAPI`): An invoke-transaction.
             send_everything (bool): ``True`` will send the difference from balance and fee.
               Defaults to ``False``.
+            private (bool): ``True`` will use the `eth_sendPrivateTransaction` RPC method.
```

Comment:
it is not necessarily true that it will use `eth_sendPrivateTransaction` RPC.  the provider plugin system allows for other options, such as non-EVM based ecosystems.

- [ ] make doc more generic here

Reply:
Probably for local/fork tests can just have it forward to sending a normal transaction, so you can validate a script that has the `private=True` kwarg locally (e.g. it basically just ignores it)

Reply:
that makes sense

Reply:
both done